src/mcts/search.cc

```diff
@@ -1322,7 +1323,8 @@ void SearchWorker::DoBackupUpdateSingleNode(
       v = n->GetQ();
       d = n->GetD();
     }
-    n->FinalizeScoreUpdate(v, d, node_to_process.multivisit);
+    n->FinalizeScoreUpdate(v / (1.0f + params_.GetShortSightedness() * depth),
```

Comment:
Any noticeable `nps` change on random backend?

Reply:
I will get those numbers, but I'm confident the answer will be no.

Reply:
10 second random benchmark with 4 threads are 340k +/- 5k on my machine for both master and this branch (with shortsightedness 0).